### PR TITLE
Added rebranding email for all users

### PR DESCRIPTION
**Pull request by ritik-bluebash**
issue #386 
Added rebranding email for all users with CTA link

Screenshot from 2022-03-16 10-28-37

**Continued image.........**
Screenshot from 2022-03-16 10-28-48

### CODE REVIEW

File: lib/tasks/email_rebranding.rake [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace :rebranding do
+  desc 'Send rebranding email to all users'
+  task email_all_users: :environment do
+    User.all.each do |user|
```
Comment (vishal-bluebash) [inline lib/tasks/email_rebranding.rake:4]: We should filter it users over here

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -8,4 +8,28 @@ def newsletter(resource_id, recipient_id)
...
   end
+
+  def email_rebranding(user)
+    session = build_passwordless_session(user)
+    session.save!
+    @current_locale = user.locale
+    @user_name = "#{user.first_name} #{user.last_name}"
+    @magic_link = send(Passwordless.mounted_as)
+                  .token_sign_in_url(session.token, subdomain: @current_locale)
+    attachments.inline['new_logo.svg'] = File.read(Rails.root.join(*%w[app javascript images new_logo.svg]))
```
Comment (vishal-bluebash) [inline app/mailers/user_mailer.rb:19]: User who is not confirmed yet will also be confirmed when they click on magic link

**Review by vishal-bluebash — CHANGES_REQUESTED** [review]
clarification  and update needed

**Comment by vishal-bluebash** [comment]
@ValerioVillani please check screenshot and verify content

**Comment by ValerioVillani** [comment]
> @ValerioVillani please check screenshot and verify content

Hi @vishal-bluebash I edited the message together with Duccio, you can find the new one here: 

https://github.com/uCV-team/ucv/issues/386

File: app/mailers/user_mailer.rb (hunk 2 above)
Comment (ritik-bluebash) [reply app/mailers/user_mailer.rb:19]: Yes sir. The user account gets confirmed, published and the user gets signed in as well to their new uCV account.

**Review by vishal-bluebash — APPROVED** [review]

**Review by ValerioVillani — APPROVED** [review]

**Pull request merged**